tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -64,30 +86,30 @@ func separateV4V6(cidrList string) (NetworkRange, NetworkRange, error) {
 	return ipv4Ranges, ipv6Ranges, nil
 }
 
-func removeOldChains(ext dep.Dependencies) {
+func (runner *Runner) removeOldChains() {
```

Comment:
@rlenglet I thought we were going to remove this method and move it to istio-iptables-cleanup?

Reply:
Looks the cleanup PR got merged. rebased and pushed another change. Resolving comment.